composer.json

```diff
@@ -20,6 +20,7 @@
   },
   "require": {
     "php": ">=5.3.3",
-    "apache/log4php": "2.3.0"
+    "apache/log4php": "2.3.0",
```

Comment:
@marcelog You're right, and I agree.

I kind of forgot about the out of the box installation. I only use PAMI as part of larger projects, in which it is integrated as a library. Thanks for mentioning the standalone use case!

For v2.0.0 I would propose using [monolog](https://github.com/seldaek/monolog) out of the box. I have a few reasons for that:

* Monolog is by far the [most used logger for PHP](https://packagist.org/search/?q=psr-3&orderBys[0][sort]=downloads&orderBys[0][order]=desc) these days.
* It fully supports PSR-3.
* It provides a staggering amount of logging protocols and services out of the box, making it very easy to log stuff just the way the user wants.

But for now, I'm happy with it as-is.